WORKSPACE

```diff
@@ -342,6 +342,22 @@ jvm_maven_import_external(
     server_urls = ["https://repo1.maven.org/maven2"],
 )
 
+jvm_maven_import_external(
+    name = "night_config_core",
+    artifact = "com.electronwill.night-config:core:3.6.5",
+    artifact_sha256 = "d47fd261e4c20c6f800a0f354644b5e83840347dde17d46537a9cb1c5a4207ba",
+    licenses = ["notice"],  # Apache 2.0
```

Comment:
It's LGPL, not Apache

https://mvnrepository.com/artifact/com.electronwill.night-config/core/3.6.5

Reply:
Updated to
```
licenses = ["notice"],  # Lesser General Public License 3.0
```

Reply:
Not that I could find. There appear to be a couple of MIT-licensed TOML libraries in the wild. I haven't tested any of them.